### PR TITLE
Improvements on load testing to avoid only hitting cache and have mor…

### PR DESCRIPTION
**Pull request by AlfredoG87**
**Description**:
Improvements on load testing to avoid only hitting cache and have more variable and randomized(ish) test data

**Related issue(s)**:

Fixes #

**Notes for reviewer**:

**Checklist**

- [ ] Documented (Code comments, README, etc.)
- [x] Tested (unit, integration, etc.)

### CODE REVIEW

File: k6/src/prepare/prep.js [hunk 1]
```diff
@@ -61,29 +61,84 @@ async function getSignedTxs(mainWallet, greeterContract) {
...
+    console.log("Main Wallet Address: " + mainWallet.address);
+    console.log("Main Wallet Initial Balance: " + ethers.utils.formatEther(await provider.getBalance(mainWallet.address)) + " HBAR");
+    const usersCount = process.env.WALLETS_AMOUNT ? process.env.WALLETS_AMOUNT : 1;
+    const contractsCount = process.env.SMART_CONTRACTS_AMOUNT ? process.env.SMART_CONTRACTS_AMOUNT : 1;
+    const smartContracts = [];
+    for (let i = 0; i < contractsCount ; i++) {
+        const contractFactory = new ethers.ContractFactory(Greeter.abi, Greeter.bytecode, mainWallet);
+        console.log(`Deploying Greeter SC  ${i}`);
+        const contract = await contractFactory.deploy("Hey World!");
+        // const receipt = await contract.deployTransaction.wait();
```
Comment (Nana-EC) [inline k6/src/prepare/prep.js:77]: nit: remove commented out lines

**Review by Nana-EC — COMMENTED** [review]
Looking good.
1 nit but also i think we should update the README to highlight the prep work carried out. 
Would be good to briefly note what contracts are deployed and the logic around wallet creation etc.
Basically an overview of the load testing logic for users

**Review by ebadiere — COMMENTED** [review]
LG

File: k6/src/prepare/prep.js (hunk 1 above)
Comment (AlfredoG87) [reply k6/src/prepare/prep.js:77]: Thanks for noticing this. 😅

Comment (AlfredoG87) [reply k6/src/prepare/prep.js]: ✅  Done

**Review by Nana-EC — APPROVED** [review]
LG

**Pull request merged**